### PR TITLE
Issue#1087: Fix filters dropdown items are not viewable

### LINKED ISSUE #1087: 🐛 [Bug report]: In mobile view, the filters are not viewable
### Descrcibe the bug?

In mobile view, the filters are not viewable and is being truncated like in the sceeenshot below

### Steps to reproduce the bug?

1. Got to this page https://reactplay.io/plays from mobile view
2. Open the filters beside the top search bar
3. Open any dropdown from the popup
4. The dropdown items are not viewable and is being truncated

### Expected behavior

I think we can make the dropdown title and dropdown in two separate line

### Desktop (Please provide your system information)

OS-Windows 11
Browser-Chrome
Version-Version-111.0.5563.115

### Mobile (Please provide your device information)

Device- Samsung
Browser-Chrome
Version-111.0.5563.115

### Screenshot / Screenshare

Screenshot_20230401-170228_Chrome

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by niteshseram**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
I have made the dropdown title and dropdown in two separate lines and also, removed some remove some padding which was a lot of spaces in the mobile view.
Also, I have made changes in the `Tube2Tunes.js` play file because the import was breaking the code because of file naming mismatch while importing.
Fixes #1087
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

image

### CODE REVIEW

File: src/plays/tube2tunes/Tube2Tunes.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Loader from './components/Loader';
 import TubeTunesLogo from './components/tubeTuneLogo.svg';
```
Comment (atapas) [inline src/plays/tube2tunes/Tube2Tunes.js:7]: @niteshseram Can you please revert this change. Its fixed separately, now this change will cause the build fail

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/plays/tube2tunes/Tube2Tunes.js (hunk 1 above)
Comment (niteshseram) [reply src/plays/tube2tunes/Tube2Tunes.js]: Reverted the change @atapas . Thanks

**Review by atapas — APPROVED** [review]
Great work @niteshseram and good to see you here as well(after the web apis project 👍 )

**Comment by niteshseram** [comment]
> Great work @niteshseram and good to see you here as well(after the web apis project 👍 )

Thanks @atapas

**Pull request merged**